torch/distributed/nn/api/remote_module.py

```diff
@@ -96,6 +99,7 @@ def __init__(
                 >>> module_cls = MyModule
             args (Sequence, optional): args to be passed to ``module_cls``.
             kwargs (Dict, optional): kwargs to be passed to ``module_cls``.
+            device (torch.device): Device where the remote node runs the user-provided module.
```

Comment:
Reword this to: `Device on the destination worker where we'd like to place this module`